middleware_auth_user_test.go

```diff
@@ -1,173 +1,285 @@
 package gimlet
 
 import (
-	"bytes"
+	"context"
```

Comment:
All the new calls to `assert.NotNil` in this file that inspect the data right after should be changed to `require.NotNil` to avoid panics

Reply:
done